CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog for Pulse3D
 =====================
 
+0.32.7 (2023-05-01)
```

Comment:
We should bump this to 0.33.0 since we're changing a bunch of metrics

Reply:
Cool I was wondering about that